src/main/java/com/nti/socialmediaappcore/payload/request/LoginRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.nti.socialmediaappcore.payload.request;
+
+import javax.validation.constraints.NotBlank;
+
+public class LoginRequest {
+    @NotBlank
+    private String username;
+
+    @NotBlank
+    private String password;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+}
```

Comment:
Use lombok annotations: https://projectlombok.org/features/all
Rename to CredentialsDTO and move to dto directory (package)